app/scripts/main.js

```diff
@@ -99,10 +95,6 @@ var iViz = (function(_, $, cbio, QueryByGeneUtil, QueryByGeneTextArea) {
       group.hasfilters = false;
       _.each(data_.groups.sample.attr_meta, function(attrData) {
         attrData.group_type = group.type;
-        if (attrData.view_type === 'table') {
```

Comment:
This to separate table data from the vue modal. This is done to reduce the number of reactive data components.